frontend/src/container/MetricsExplorer/MetricDetails/MetricDetails.tsx

```diff
@@ -131,8 +131,8 @@ function MetricDetails({
 						>
 							Open in Explorer
 						</Button>
-						{/* Show the based on the feature flag. Will remove before releasing the feature */}
-						{showInspectFeature && (
+						{/* Show the inspect button if the metric type is GAUGE */}
```

Comment:
This comment seems to be at the wrong place

Reply:
Actually it's at the intended place. `showInspectFeature` checks if the metric type is gauge